src/js/components/TabCreateSpace.js

```diff
@@ -0,0 +1,85 @@
+/* @flow */
+import {useDispatch} from "react-redux"
+import React, {useState} from "react"
+import styled from "styled-components"
+
+import type {Styled} from "../types/styled"
+import {createSpace} from "../flows/createSpace"
+import ErrorFactory from "../models/ErrorFactory"
+import FileInput from "./common/forms/FileInput"
+import InputField from "./common/forms/InputField"
+import InputLabel from "./common/forms/InputLabel"
+import Notice from "../state/Notice"
+import PrimaryButton from "./common/buttons/PrimaryButton"
+import SelectInput from "./common/forms/SelectInput"
+import TextInput from "./common/forms/TextInput"
+
+const Wrap: Styled<> = styled.section`
+  max-width: 460px;
+  width: 100%;
+  margin: 0 auto;
+`
+
+const FormWrap = styled.form`
+  .input-field {
+    margin-bottom: 18px;
+  }
+`
+
+const SubmitWrap = styled.div`
+  margin-top: 18px;
+  display: flex;
+  justify-content: flex-end;
+`
+
+export default function TabCreateSpace() {
```

Comment:
This is a great new component and flow for space creation! Why don't combine the import files flow in here as well? We could still provide defaults for the name so that the user still only needs to import the logs, click `create` and go.

Reply:
In the future, I think an approach like you suggested here will make a lot of sense. Since it's a different scope that the "index search" feature, we'll address it in the future.